### PR TITLE
Streamlined UI

### PR DESCRIPTION
**Pull request by microfauna**
- Clean up homepage search to be clearer and easier to use
- Add better login state transitions / disable input when loading
- Clean up media query breakpoints for better scaling.
- This may conflict with #49

### CODE REVIEW

File: src/styles/screen.scss [hunk 1]
```diff
@@ -59,13 +59,13 @@ main {
   box-shadow: none !important;
   margin-top: -1px;
   margin-left: -1px;
-  width: 100%;
+  width: calc(100% + 2px) !important;
```
Comment (microfauna) [inline src/styles/screen.scss:62]: Makes the dropdown for tags, etc line up with the search box

File: src/styles/screen.scss [hunk 2]
```diff
@@ -89,10 +89,6 @@ main {
   }
 }
 
-.login-screen {
-  min-height: calc(100vh - 200px)
```
Comment (microfauna) [inline src/styles/screen.scss:93]: Not sure this was needed

File: src/views/all/Home.vue [hunk 3]
```diff
@@ -2,104 +2,73 @@
   <main>
     <div>
       <div
-        ref="hidebar"
```
Comment (microfauna) [inline src/views/all/Home.vue:5]: Tons of changes simplifying the flow and media queries.

File: src/views/owner/EditBusiness.vue [hunk 4]
```diff
@@ -4,7 +4,7 @@
       <h1 class="text-2xl">Edit Business</h1>
       <router-link to="/owner" class="border border-gray-600 px-6 py-2 rounded inline-block mb-2">Go Back</router-link>
     </section>
-    <div class="flex flex-col md:flex-row border border-gray-600 mb-6 shadow rounded">
+    <div class="flex flex-col lg:flex-row border border-gray-600 mb-6 shadow rounded">
```
Comment (microfauna) [inline src/views/owner/EditBusiness.vue]: Just two major breakpoints now, since tablet scales down nicely.

File: src/views/all/Home.vue [hunk 5]
```diff
@@ -2,121 +2,95 @@
...
+                  v-for="(nation, i) in search_affiliations"
+                  :key="i"
+                >
+                  <span class="text-gray-500">{{nation.name}}</span>
+                  <span v-if="i + 1 < search_affiliations.length - 1">
+                    ,&nbsp;
+                  </span>
+                  <span v-if="i + 1 === search_affiliations.length - 1">
+                    &nbsp;or&nbsp;
+                  </span>
```
Comment (arecvlohe) [inline src/views/all/Home.vue:73]: Might be helpful to name these expressions, like `hasAffiliations` or something

File: src/views/all/Home.vue [hunk 6]
```diff
@@ -179,15 +137,19 @@ export default {
...
         return getBusinessQuery(this.search_query, this.search_affiliations, this.search_tags)
       },
       variables () {
-        const query = this.search_query || undefined
+        let query = this.search_query || undefined
         const affiliations = this.search_affiliations.length > 0 ? this.search_affiliations.map(a => a.id) : undefined
         const tags = this.search_tags.length > 0 ? this.search_tags.map(t => t.id) : undefined
 
-        return this.minimum_search_criteria ? { query, affiliations, tags } : {}
+        if (query) query = query.trim().replace(/\s/g, '|')
```
Comment (arecvlohe) [inline src/views/all/Home.vue]: I think you can keep this as `const` and apply the trim and replace to the search query

**Review by arecvlohe — DISMISSED** [review]
Nothing blocking really

**Comment by arecvlohe** [comment]
Do you think we can add a rule to allow login to `https://deploy-preview-50--native-owned.netlify.app/` for only my email? Or a test email? It would be cool to test out more functionality and document changes using Netlify Drawer

**Comment by arecvlohe** [comment]
recording

Kind of a jarring thing that happens when you start to type it begins pulling down all search results it appears


<details><summary>Browser metadata</summary>





```
Path:      /
Browser:   Firefox 90.0 on Mac OS 10.15
Viewport:  1440 x 792 @2x
Language:  en-US
Cookies:   Enabled
```

[View on BrowserStack](https://live.browserstack.com/dashboard#url=https%3A%2F%2Fdeploy-preview-50--native-owned.netlify.app%2F&os=OS+X&os_version=Catalina&browser=Firefox&browser_version=90.0&resolution=1440x792&speed=1&start=true)



</details>

[Open Deploy Preview](https://deploy-preview-50--native-owned.netlify.app/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=60b44fe750e0e400088efc53&commentID=60b5345d6a1bd9c45d2f6240&resolution=resolved)

**Comment by arecvlohe** [comment]
recording

I want to be able to find natives in tech but it appears that I can't for some reason. Not sure what the issue might be


<details><summary>Browser metadata</summary>





```
Path:      /
Browser:   Firefox 90.0 on Mac OS 10.15
Viewport:  1440 x 792 @2x
Language:  en-US
Cookies:   Enabled
```

[View on BrowserStack](https://live.browserstack.com/dashboard#url=https%3A%2F%2Fdeploy-preview-50--native-owned.netlify.app%2F&os=OS+X&os_version=Catalina&browser=Firefox&browser_version=90.0&resolution=1440x792&speed=1&start=true)



</details>

[Open Deploy Preview](https://deploy-preview-50--native-owned.netlify.app/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=60b44fe750e0e400088efc53&commentID=60b5349cc99fd9f4566e2ed9&resolution=resolved)

**Comment by arecvlohe** [comment]
screenshot

Kind of a weird focus state for the search input compared to the other inputs


<details><summary>Browser metadata</summary>





```
Path:      /
Browser:   Firefox 90.0 on Mac OS 10.15
Viewport:  1440 x 792 @2x
Language:  en-US
Cookies:   Enabled
```

[View on BrowserStack](https://live.browserstack.com/dashboard#url=https%3A%2F%2Fdeploy-preview-50--native-owned.netlify.app%2F&os=OS+X&os_version=Catalina&browser=Firefox&browser_version=90.0&resolution=1440x792&speed=1&start=true)



</details>

[Open Deploy Preview](https://deploy-preview-50--native-owned.netlify.app/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=60b44fe750e0e400088efc53&commentID=60b5351a4a73d24915a8c47f&resolution=resolved)

**Comment by arecvlohe** [comment]
Not saying we should do this now but it might be cool to allow clicking on tags to update the tags in the search bar. So if I see something I like I can check out quickly others businesses that do the same kind of thing
<details><summary>Browser metadata</summary>





```
Path:      /
Browser:   Firefox 90.0 on Mac OS 10.15
Viewport:  1440 x 792 @2x
Language:  en-US
Cookies:   Enabled
```

[View on BrowserStack](https://live.browserstack.com/dashboard#url=https%3A%2F%2Fdeploy-preview-50--native-owned.netlify.app%2F&os=OS+X&os_version=Catalina&browser=Firefox&browser_version=90.0&resolution=1440x792&speed=1&start=true)



</details>

[Open Deploy Preview](https://deploy-preview-50--native-owned.netlify.app/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=60b44fe750e0e400088efc53&commentID=60b5355b5fa6c4527b253902&resolution=resolved)

**Comment by microfauna** [comment]
> I want to be able to find natives in tech but it appears that I can't for some reason. Not sure what the issue might be
> Browser metadata

Fixed this!

File: src/views/all/Home.vue (hunk 6 above)
Comment (microfauna) [reply src/views/all/Home.vue]: Ah right, it should remain an empty string. Done!

**Comment by microfauna** [comment]
> Do you think we can add a rule to allow login to `https://deploy-preview-50--native-owned.netlify.app/` for only my email? Or a test email? It would be cool to test out more functionality and document changes using Netlify Drawer

Let's create a ticket and put it in the backlog, I can get on it soon

**Comment by arecvlohe** [comment]
> https://deploy-preview-50--native-owned.netlify.app/

https://github.com/nativesintech/native-owned-businesses/issues/54

**Review by arecvlohe — DISMISSED** [review]

**Pull request merged**